.gitmodules

```diff
@@ -1,7 +1,3 @@
-[submodule "ros/src/msgs/platform_automation_msgs"]
-	path = ros/src/msgs/platform_automation_msgs
-	url = https://github.com/astuff/platform_automation_msgs.git
-	branch = 51ceed71c28773887648993ceec803ca498698ae
 [submodule "ros/src/sensing/drivers/lidar/packages/robosense"]
 	path = ros/src/sensing/drivers/lidar/packages/robosense
 	url = https://github.com/CPFL/robosense
```

Comment:
```suggestion
	url = https://github.com/CPFL/robosense
[submodule "automotive_autonomy_msgs"]
        path = automotive_autonomy_msgs
        url = git@github.com:astuff/automotive_autonomy_msgs.git
```

Reply:
Now that these packages have been released into the ROS index, the following will install the binary packages so we no longer need the submodule:

```
rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO
```

Reply:
I got it! In my environment, I need `rosdep update` once. Greatly thanks.